aura/lib/Aura/Languages.hs

```diff
@@ -422,6 +441,7 @@ depError l (NonExistant (PkgName s) (PkgName par)) = case l of
   Italian    -> "Non è stato possibile trovare la dipendenza " <> bt s <> "."
   Esperanto  -> "La dependeco " <> bt s <> " de " <> bt par <> " ne povis troviĝi."
   Dutch      -> "Het afhankelijkheid " <> bt s <> "kan niet worden gevonden."
+  Ukrainian  -> "Залежність " <> bt s <> "не було знайдено."
```

Comment:
This phrase looks inconsistent about handling spaces, but probably " space" too?